### PR TITLE
revert some switches back to checkboxes

### PR DESCRIPTION
**Pull request by michaelkirk**
partial revert of 90bb4ac0

In many ways switches and checkboxes seem interchangeable, but in certain
contexts one may be more appropriate.

For an overview that I mostly agree with:
https://uxplanet.org/checkbox-vs-toggle-switch-7fc6e83f10b8

**before**
Screen Shot 2020-07-28 at 1 48 47 PM

**after**
Screen Shot 2020-07-28 at 1 49 30 PM

Most things continue to use switches - especially those which produce instance feedback without a "submit" step.

Screen Shot 2020-07-28 at 1 50 26 PM

However, I feel like the switch labels are too large. That didn't change with this PR, but would you be interested in a followup PR which makes the switch text size comparable to what you see on the checkboxes?

### CODE REVIEW

**Comment by dabreegster** [comment]
Screenshot from 2020-07-28 13-31-06
DPI / `ctx.get_scale_factor()` is my nemesis forever. `Btn::custom` scales everything up, which is why everything is huge. I have a bad habit of not regularly testing things with a second screen. I can poke at this soon, unless you want to.

File: ezgui/examples/demo.rs [hunk 1]
```diff
@@ -259,8 +259,8 @@ fn make_controls(ctx: &mut EventCtx) -> Composite {
             .named("paused"),
             Btn::text_fg("Reset timer").build(ctx, "reset the stopwatch", None),
             Btn::text_fg("New faces").build(ctx, "generate new faces", hotkey(Key::F)),
-            Checkbox::text(ctx, "Draw scrollable canvas", None, true),
-            Checkbox::text(ctx, "Show timeseries", lctrl(Key::T), false),
+            Checkbox::checkbox(ctx, "Draw scrollable canvas", None, true),
```
Comment (dabreegster) [inline ezgui/examples/demo.rs:262]: `switch` -- these have immediate effect. `cd ezgui; cargo run --example demo`

File: game/src/layer/pandemic.rs [hunk 2]
```diff
@@ -231,7 +231,8 @@ fn make_controls(
         ]),
     ];
 
-    col.push(Checkbox::text(
+    // MJK: how to show this UI?
```
Comment (dabreegster) [inline game/src/layer/pandemic.rs:234]: Ooh, it's undocumented. Run with `--pandemic`, then it'll show up in layers. This was some work by @omalaspinas that's on hold.

File: ezgui/examples/demo.rs (hunk 1 above)
Comment (michaelkirk) [reply ezgui/examples/demo.rs]: done!

File: game/src/layer/pandemic.rs (hunk 2 above)
Comment (michaelkirk) [reply game/src/layer/pandemic.rs]: done!

**Comment by dabreegster** [comment]
Thanks for the PR!

**Comment by michaelkirk** [comment]
> Also, I wonder what's possibly going on with the gradient...

Do you mean how ours are different colors? Maybe related to you looking at a population of zero?

**Comment by dabreegster** [comment]
> Do you mean how ours are different colors? Maybe related to you looking at a population of zero?

The labels change with population, but the scale itself should be fixed:

https://github.com/dabreegster/abstreet/blob/3879753cd2378b01ecda1aefe7b15ce71d487308/game/src/common/colors.rs#L121

It's also caused by DPI. If I run with `--scale_factor=2`, I see the same thing.

**Pull request merged**